### DISCUSSION TITLE
Do I need to know Python before learning Manim?

### DISCUSSION THREAD

**Discussion by Idealtic** [ManimCommunity/manim#4033] [Q&A] [answered]
I'm completely new to Python and I want to learn Manim along with Python as well. Is this possible? Do I need to learn Python before getting started with Manim?

**Comment by behackl** [comment, marked as answer]
Personally I think that you might be able to do simple things with Manim without having any deeper knowledge of Python -- but you will reach your limits quite quickly. I'd rather recommend working through some simple Python tutorial first, and then diving deeper into Manim.
